frontend/dist/shotgrid-addon.js

```diff
@@ -236,19 +241,20 @@ const getAyonUsers = async () => {
 }
 
 
-const createNewUserInAyon = async (login, email, name) => {
+const createNewUserInAyon = async (user_name, login, email, name) => {
   /* Spawn an AYON Event of topic "shotgrid.event" to synchcronize a project
   from Shotgrid into AYON. */
   call_result_paragraph = document.getElementById("call-result");
 
   response = await ayonAPI
-    .put("/api/users/" + login, {
-      "active": true,
+    .put("/api/users/" + user_name, {
       "attrib": {
         "fullName": name,
         "email": email,
       },
-      "password": login,
```

Comment:
The PR description does not mention the removal of the `password` transfer. Is this intentional?

Reply:
and yeah same question, what is the password by default otherwise?

Reply:
This is not password transfer. There is pseudo password creation, this need to be changed by user anyway.

Reply:
how does the user know their first password on creation to login so they can change it?